apps/emqx_rule_engine/test/emqx_rule_engine_api_SUITE.erl

```diff
@@ -281,3 +263,45 @@ test_rule_params(Sql, Payload) ->
             <<"sql">> => Sql
         }
     }.
+
+t_rule_engine(_) ->
+    _ = simple_rule_fixture(),
+    {200, Config} = emqx_rule_engine_api:'/rule_engine'(get, #{}),
+    ?assert(not maps:is_key(rules, Config)),
+    {200, #{
+        jq_function_default_timeout := 12000
+        % hidden! jq_implementation_module := jq_port
```

Comment:
Q: is it really hidden?  When I do `emqx_config:get([node])`, for example, I still get hidden fields in the answer :thinking: 
Or does the swagger API / minirest filter those out based on the schema?

Reply:
Are you sure about getting it? Because that’s why I had to remove it. I’m confused.

Reply:
Yes, it’s not returned.